src/m_misc.c

```diff
@@ -406,6 +407,13 @@ default_t defaults[] = {
     "number of sound effects handled simultaneously"
   },
 
+  {
+    "snd_resampler",
+    (config_t *) &snd_resampler, NULL,
+    {.s = "linear"}, {0}, string, ss_gen, wad_no,
+    "OpenAL resampler (\"nearest\", \"linear\"(default), \"cubic\")"
```

Comment:
@fabiangreffrath OpenAL Soft report 7 resamplers on my system:
```
Nearest
Linear
Cubic
11th order Sinc (fast)
11th order Sinc
23rd order Sinc (fast)
23rd order Sinc
```
In order to properly present it, we need a menu option that we would populate dynamically. Although, I don't like another menu item for this.

Reply:
I am fine with the first three, and also with not having this choice in the menu. The default is the same as before now, and the effort to change it, too.

Reply:
Personally, I might even stick with cubic. It has a bit more "woom" than linear without sounding as scratchy. The sinc modes sound too different to me.

Reply:
To be honest I don't hear much difference 😆

Reply:
But what is closer to old Woof or Crispy sound? It's tempting to remove all of our printing and resampler settings, it just seems confusing to users, and OpenAL Soft knows best anyway.

Reply:
To me Crispy with `use_libsamplerate` set to `1`, latest Woof with `snd_resampler` set to `linear`, and Woof 10.5.1 all sound indistinguishable from one another.

Reply:
> To me Crispy with `use_libsamplerate` set to `1`, latest Woof with `snd_resampler` set to `linear`, and Woof 10.5.1 all sound indistinguishable from one another.

Confirmed.